### DISCUSSION TITLE
Reorder GUI to seperate "per profile settings/tasks" and "global settings/tasks"

### DISCUSSION THREAD

**Discussion by deermo** [borgbase/vorta#814] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently, the GUI mixes elements of per profile settings (i.e. the tabs "Repository", "Sources", ...) and global settings / tasks (i.e. "Misc" tab and the "Start backup" / progress bar area). 

**Describe the solution you'd like**
Rearranging the GUI so that there are two horizontally seperated areas. The upper one could include to profile switch  and the 4 tabs ("Repository", "Sources", ...). The lower one could also have two tabs, with the first beeing "Current task" (maybe indicating which profile is running, #578), and the second beeing the current "Misc" tab (i.e. namend "Global options")? 

**Describe alternatives you've considered**
Renaming "Misc" to "Global options" could also help.

**Comment by samuel-w** [comment]
Changing "Misc" to "Global Settings" would be a more accurate name for it.

The backup bar is already separated by the line of the tab box.
![colours](https://user-images.githubusercontent.com/67625312/92215026-d3bb7800-ee59-11ea-91c0-a74c6def6b86.png)

Separating the UI boxes would take too much screen space. We could visually separate the tab buttons without splitting the box https://stackoverflow.com/questions/37201443/move-two-qt-tabs-to-the-right-leave-rest-to-the-left/37203075
